controller/src/main/java/org/jboss/as/controller/SimpleMapAttributeDefinition.java

```diff
@@ -125,9 +127,9 @@ public SimpleMapAttributeDefinition build() {
             if (elementValidator == null) {
                 elementValidator = new ModelTypeValidator(valueType, allowNull, allowExpression);
             }
-            if (attributeMarshaller == null) {
+            /*if (attributeMarshaller == null) {
                 attributeMarshaller = new MapAttributeMarshaller();
-            }
+            }*/
```

Comment:
I'm not sure how the change at L108 ties in to the rest of this PR either. It looks like the idea here is SimpleMapAttributeDefinition.MapAttributeMarshaller is meant to no longer be used (since the change here comments out the only use) with replacement by AttributeMarshallers.PropertiesAttributeMarshaller and a parser change too.